include/connection.rb

```diff
@@ -46,6 +46,8 @@ def sock_puts(str)
   def write(str)
     reconnect unless socket_working?
 
+    puts "Http2: Writing headers: #{str}" if @debug
```

Comment:
Isn't `write` used to write everything and not just headers?

Somehow I think that maybe a debug level would be better. Level 1 would write headers. Level 2 would also write the body. What do you think?